pkg/operator/controller/nodeobservability/daemonset_util.go

```diff
@@ -11,6 +11,12 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// LabelsForNodeObservability returns the labels for selecting the resources
+// belonging to the given node observability CR name.
+func LabelsForNodeObservability(name string) map[string]string {
```

Comment:
nit: I would still prefer this to be private and explicitly added in the test just to decouple them.

Reply:
Done